### DISCUSSION TITLE
Custom Color Background

### DISCUSSION THREAD

**Discussion by Rodger2000** [ahrm/sioyek#1222] [Ideas]
In the dark mode and normal color mode, background color controls the color behind/outside the "page" of the document. However in custom color mode, background color refers to both the color of the page and the color behind it.

Would it be possible to control these two options separately in custom color mode?

**Comment by ahrm** [comment]
We have a `custom_color_mode_empty_background_color` config for that.

**Comment by Rodger2000** [comment]
Sorry I can't seem to get this working. I have verified I am on 2.0 and my whole config is just:

custom_color_mode_empty_background_color 1.0 0.0 0.0 
custom_background_color              0.14 0.16 0.18
custom_text_color                    0.83 0.78 0.67 
startup_commands             toggle_custom_color;toggle_statusbar

The background empty color is just defaulting to the custom_background_color unlike of what i see #1193.

**Comment by ahrm** [comment]
I think this option was added in the development branch and is not available in the 2.0 release.
